packages/parser/src/language/info/index.ts

```diff
@@ -0,0 +1,15 @@
+export * from './infoModule.js';
+export * from './infoParser.js';
+export * from './infoTokenBuilder.js';
```

Comment:
Remove if possible.

Reply:
Where is the module used?

Reply:
In creating services to get parser, we need it for `tests` dir.